package/MDAnalysis/core/topologyattrs.py

```diff
@@ -265,6 +265,25 @@ def _attach_transplant_stubs(attribute_name, topology_attribute_class):
                 setattr(dest_class, method_name, stub)
 
 
+# TODO: remove bfactors in 3.0
+BFACTOR_WARNING = ("The bfactor topology attribute is only "
```

Comment:
This should be documented somewhere.

Reply:
I'm not sure we really document all the topology attributes anywhere. I added it as a docstring to the bfactor/s properties?

Reply:
Yeah that sounds reasonable, for some reason I thought we had a big table somewhere, but maybe I'm getting confused with something else.

Reply:
in `__init__.py`, but I think that might firstly be woefully out of date, and secondly never mentioned bfactors to start off with :p